src/main/java/com/epam/tc/nitcenkov/hw9/service/BoardService.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.nitcenkov.hw9.service;
+
+import com.epam.tc.nitcenkov.hw9.dto.BoardDTO;
+import com.google.gson.GsonBuilder;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import java.util.Map;
+
+public class BoardService extends BasicService {
+
+    private static final String GET_OR_UPDATE_OR_DELETE_BOARD = "1/boards/";
+    private static final String CREATE = "1/boards";
+
+    public static BoardDTO readBoardFromResponse(Response response) {
```

Comment:
same code in 3 files, pls update to have only one method

 return new GsonBuilder().excludeFieldsWithModifiers().create()
                                .fromJson(response.getBody().asString(),
                                    BoardDTO.class);